### ISSUE TITLE
Online editor spec after 3 indentations not visible with screen reader

### ISSUE THREAD

**Issue by thomcsmits** [gosling-lang/gosling-website#59]
When accessing the online editor fully through screen reader and keyboard, everything in the spec after 3 indentations is shown as 'blank'.
